app/services/ClickHandler.coffee

```diff
@@ -0,0 +1,69 @@
+$ = require 'jquery'
+
+class ClickHandler
+
+  constructor: (@board, two, @clicked=[]) ->
+    @numClicked = @clicked.length
+    return unless @board.cells?
+    for row in @board.cells
+      break if row.length is 0
```

Comment:
we were trying to avoid breaks although it could be just a tiny faster. You can do 
```
unless row.length is 0
```